docs/section16/Rule16-1.md

```diff
@@ -0,0 +1,47 @@
+# Elevators – Rule 16-1  
+**Schema Version:** 0.0.36        
+**Mandatory Rule:** True          
+**Rule ID:** 16-1  
+**Rule Description:** The elevator peak motor power shall be calculated according to the equation in Table G3.1-16  
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** G3.1  
+**Appendix G Section Reference:** Table G3.1-16 Baseline Building Performance
+**Data Lookup:** Table G3.9.1, Table G3.9.2, Table G3.9.3  
+**Evaluation Context:** Each elevator  
+
+**Applicability Checks:**  
+  1. The proposed building has an elevator.  
+
+**Function Call:**  
+data_lookup
+
+**Applicability Check 1:**
+- Rule is applicable if any building in the proposed RMD has an elevator: `if any(len(building.elevators) > 0 for building in P_RMD...buildings:`
+
+## Rule Logic:
+- For each elevator in the building: `for elevator in B_RMD...elevators`
+  - Get the number of floors served by the elevator: `total_floors_served_b = elevator.number_of_floors_served` 
+  - Get the elevator motor power: `elevator_motor_power_b = elevator.motor_power`
+  - Get the elevator cab weight: `elevator_cab_weight_b = elevator.cab_weight`
+  - Get the elevator cab counterweight: `elevator_cab_counterweight_b = elevator.cab_counterweight`
+  - Get the elevator design load: `elevator_design_load_b = elevator.design_load`
+  - Get the elevator speed: `elevator_speed_b = elevator.speed`
+  - If any detailed elevator data parameters are not defined: `if any(param == Null for param in [elevator_motor_power_b, elevator_cab_weight_b, elevator_cab_counterweight_b, elevator_design_load_b, elevator_speed_b]): is_undetermined = true`
+  - Get the elevator mechanical efficiency: `elevator_mechanical_efficiency_b = data_lookup('table_g3.9.2', total_floors_b)['mechanical_efficiency']`
+  - Calculate the motor brake horsepower: `motor_brake_horsepower_b = (elevator_cab_weight_b + elevator_design_load_b - elevator_cab_counterweight_b) * elevator_speed_b / 33000 / elevator_mechanical_efficiency_b`
+  - If the total number of floors is greater than 4: `if total_floors_b > 4:`
```

Comment:
looks like the variable name is inconsistent with above total_floors_served_b verses total_floors_b